x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/constants/archives_metadata.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import archivesMetadata from '../../../../../../apm_api_integration/common/fixtures/es_archiver/archives_metadata';
+
+export default archivesMetadata;
```

Comment:
Not sure if this barrel file is providing us any value, I would import archives metadata directly

Reply:
My goal when doing this was to avoid having to modify multiple imports if we eventually move `archivesMetadata` to the deployment-agnostic folder.